flink-core/src/main/java/org/apache/flink/configuration/TaskManagerOptions.java

```diff
@@ -707,6 +707,12 @@ public class TaskManagerOptions {
                                                     "The %s mode tries to spread out the slots evenly across all available %s.",
                                                     code(TaskManagerLoadBalanceMode.SLOTS.name()),
                                                     code("TaskManagers")),
+                                            text(
+                                                    "The %s mode tries to schedule evenly all tasks based on its' number across all available %s. "
```

Comment:
I don't understand this sentence. I am not sure what "schedule evenly all tasks based on its' number" means.

Reply:
The meaning expressed here is that the scheduler will try its best to ensure that the scheduling results are concentrated and the number of tasks on each Taskmanager is similar.
I will add a more detailed description